tests/server/api/mutations/test_dataset_mutations.py

```diff
@@ -250,3 +251,35 @@ async def spans(session):
             cumulative_llm_token_count_completion=0,
         )
     )
+
+
+async def test_delete_a_dataset(
+    session,
+    test_client,
+    empty_dataset,
+):
+    dataset_id = GlobalID(type_name="Dataset", node_id=str(1))
+    mutation = textwrap.dedent(
+        """
+        mutation ($datasetId: GlobalID!) {
+          deleteDataset(input: { datasetId: $datasetId })
+        }
+        """
+    )
```

Comment:
```suggestion
    mutation = (
        """
        mutation ($datasetId: GlobalID!) {
          deleteDataset(input: { datasetId: $datasetId })
        }
        """
    )
```

I believe GraphQL ignores whitespace.